Update docs for assigning path to all_video_file_paths

hom-bahrani opened the pull request:
# What does this PR do?

Updating a few lines in the video classification tasks guide as the way its written it seems like we are not actually iterating over the file paths, but rather the string. 

Fixes # (issue)


## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

hom-bahrani committed:
Update docs for assigning path to all_video_file_paths

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22522). All of your documentation changes will be reflected on that endpoint.

sgugger commented on docs/source/en/tasks/video_classification.mdx:
```diff
@@ -127,7 +127,12 @@ Next up, you will derive the set of labels present in the dataset. Also, create
->>> class_labels = sorted({str(path).split("/")[2] for path in all_video_file_paths})
+>>> from pathlib import Path
+
+>>> base_path = Path("/path/to/UCF101_subset")
```
Should reuse the `file_path` above here.

hom-bahrani replied:
well spotted

sgugger reviewed (commented):
Thanks for your PR! I think we can fix the sample without needing the user to input anything.

hom-bahrani committed:
reuse file_path

sgugger reviewed (approved):
LGTM, thanks! @amyeroberts could you have a quick second look?

amyeroberts commented on docs/source/en/tasks/video_classification.mdx:
```diff
@@ -127,7 +127,12 @@ Next up, you will derive the set of labels present in the dataset. Also, create
->>> class_labels = sorted({str(path).split("/")[2] for path in all_video_file_paths})
+>>> from pathlib import Path
+
+>>> base_path = Path(file_path)
```
I think this might have to be `UCF101_subset`. The dataset when downloading with `hf_hub_download` I believe will download to the cache folder -- `file_path` -- whereas the tar files will be extracted to the local directory.

amyeroberts reviewed (approved):
Thanks for updating this! 🙏 

Just a comment on the file path for the `base_path` object. Have you run this code locally or on a collab to confirm it runs?

hom-bahrani committed:
update base_path to UCF101_subset

hom-bahrani commented:
@amyeroberts yes you're right, the dataset is downloaded to the .cache directory and extracted into local directory. I have updated accordingly and tested in colab to confirm that its working 

<img width="1387" alt="Screenshot 2023-04-05 at 16 06 57" src="https://user-images.githubusercontent.com/8465628/230124560-1eef5951-70f3-4b10-b4a8-4d1766bcb531.png">

amyeroberts reviewed (approved):
Thanks for checking and updating! 

It seems there is an issue with your CircleCI permissions, as the tests won't run.
Could you try refreshing your permissions as shown [here](https://support.circleci.com/hc/en-us/articles/360048210711-How-to-Refresh-User-Permissions-)?
